### PR TITLE
auth password can be set on a ThreadwisePool

### PR DESCRIPTION
**Pull request by myGuruPitka**
auth will be called once per client connection

### CODE REVIEW

File: src/Sider/RedisSettings.cs [hunk 1]
```diff
@@ -61,6 +62,7 @@ private RedisSettings()
       // set defaults
       Host = "localhost";
       Port = 6379;
+      Password = string.Empty;
```
Comment (chakrit) [inline src/Sider/RedisSettings.cs]: `null` here would be nicer i think.

Comment (myGuruPitka) [inline src/Sider/RedisSettings.cs]: iv'e changed the default password to null.

**Comment by chakrit** [comment]
@myGuruPitka I just saw that this is only for the `ThreadwisePool` but `RedisSettings` is actually used for all clients, so adding a `Password` there might be a bit misleading.

If you'd be so kind, I need one PR that do this on the client itself, instead of at the pool. : )

**Comment by chakrit** [comment]
Available in [v0.10.0](https://www.nuget.org/packages/Sider/0.10.0)

**Comment by myGuruPitka** [comment]
I'll be happy to. 
Could you please explain where should the Auth call to be executed?

**Pull request merged**